Userland/Applications/Dictionary/Dictionary.cpp

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (c) 2021, Robin Allen <r@foon.uk>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <fcntl.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include <AK/StringBuilder.h>
+
+#include "Dictionary.h"
+
+namespace Dictionary {
+
+static char const* part_of_speech_name(char pos)
+{
+    switch (pos) {
+    case 'n':
+        return "noun";
+    case 'v':
+        return "verb";
+    case 's':
+    case 'a':
+        return "adjective";
```

Comment:
Please make this helper return a StringView and use StringView literals

```c++
static StringView part_of_speech_name(char pos)
{
    switch (pos) {
    case 'n':
        return "noun"sv;
    case 'v':
        return "verb"sv;
     ...
}
```

Reply:
Done.